src/cli/node/json_rpc_client.rs

```diff
@@ -20,29 +19,33 @@ use witnet_crypto::{
     hash::calculate_sha256,
     key::{CryptoEngine, ExtendedPK, ExtendedSK},
 };
+
 use witnet_data_structures::{
     chain::{
+        priority::{PrioritiesEstimate, Priority, PriorityEstimate, TimeToBlock},
+        tapi::{current_active_wips, ActiveWips},
         Block, ConsensusConstants, DataRequestInfo, DataRequestOutput, Environment, Epoch,
         Hashable, KeyedSignature, NodeStats, OutputPointer, PublicKey, PublicKeyHash, StateMachine,
         SupplyInfo, SyncStatus, ValueTransferOutput,
     },
-    mainnet_validations::{current_active_wips, ActiveWips},
     proto::ProtobufConvert,
-    transaction::Transaction,
+    transaction::{DRTransaction, Transaction, VTTransaction},
     transaction_factory::NodeBalance,
+    types::SequentialId,
```

Comment:
This is actually not needed for the node CLI because that JSON-RPC client does not support sending multiple messages in parallel. The id is only useful if we send two messages without waiting for the response of the first one, and we never do that here so the id is always ignored. We should change this client for a better one some day, and then the id generation will be part of the library, hopefully.

Reply:
You are right. I only implemented this under the assumption that a proper JSON-RPC server wouldn't allow static IDs. I understand that this has no real impact in our case. Do you think we should ditch this, or leave it as is for the sake of protocol politeness? (taking into account that the cost of the sequential ID in this case is negligible as it just increments a number and resides entirely on the client)

Reply:
I would remove it because it is not needed, and removing it makes the code simpler.